app/helpers/header_helpers.py

```diff
@@ -1,6 +1,13 @@
-def get_span_and_trace(headers):
+from typing import Union
+
+from werkzeug.datastructures import EnvironHeaders
+
+
+def get_span_and_trace(
+    headers: EnvironHeaders,
+) -> Union[tuple[None, None], tuple[str, str]]:
     try:
-        trace, span = headers.get("X-Cloud-Trace-Context").split("/")
+        trace, span = headers.get("X-Cloud-Trace-Context").split("/")  # type: ignore
```

Comment:
Ignore only needed because this is using `EAFP`, ignore can be removed if we did `LBYL`.  Reference: https://devblogs.microsoft.com/python/idiomatic-python-eafp-versus-lbyl/